virttest/lvm.py

```diff
@@ -634,7 +634,7 @@ def make_emulate_image(self):
         """
         Create emulate image via dd with 8M block size;
         """
-        img_size = self.params["lv_size"]
+        img_size = normalize_data_size(self.params["lv_size"])
```

Comment:
Sorry, I forget that `params["lv_size"]` has been `normalize_data_size` in `__format_params`.
Please refer:
```
class LVM(object):
...
   def __init__(self, params):
        self.params = self.__format_params(params)
        ...
    def __format_params(self, params):
        lv_size = params.get("lv_size")
        if lv_size is None:
            lv_size = params["image_size"]
        params["lv_size"] = normalize_data_size(lv_size)
```
so I think we don't need to `normalize_data_size` again.

Reply:
updated